src/components/Layout.jsx

```diff
@@ -34,6 +56,18 @@ export function Layout({ children }) {
 					combined with other information or when used to build a profile of an
 					individual, even if that individual's name is unknown.
 				</p>
+        
+        <div className='flex w-full justify-between items-center mt-8'>
+					<p>
+						{theme !== 'dark' ? 'Light Mode' : 'Dark Mode'}
```

Comment:
Given the semantics and design attributes of the `Toggle` component, changing the text would confuse users. It'll also possibly confuse users like what happened with the IPhone's label-changing toggles recently.